src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File mainFile = new File(fromFileName);
+        String[] words = readFromFile(mainFile).split("\\W+");
+
+        int sumOSupply = 0;
+        int buyOSupply = 0;
+
+        for (int i = 0; i < words.length; i += 2) {
+            if (words[i].equals(SUPPLY)) {
+                sumOSupply += Integer.parseInt(words[i + 1]);
+            } else if (words[i].equals(BUY)) {
```

Comment:
I think we can remove `else` or `if` here.
If we assume that there will only be `supply` or `buy`, then we need to remove the `if`, if there may be other values, then we need to remove the `else` (it will be safer).

Reply:
Also avoid `Integer.parseInt(words[i + 1]);` duplication.